lib/TickLib.sol

```diff
@@ -483,7 +474,6 @@ library FieldLib {
 
   // Will throw if field is empty
   function firstOnePosition(Field field) internal pure returns (uint) {
-    // FIXME stop checking for 0 or integrate it into ctz function in assembly
     unchecked {
       require(!field.isEmpty(),"field is 0");
```

Comment:
Can we remove the require? I'd expect that we should be able to

Reply:
fixed in 962779950c4407e50d1a2c9882e0f1ff6bfb4d56

however all code that calls `[first|last]OnePosition` should be wary about this